<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_aparteman</class>
 <widget class="QDialog" name="show_aparteman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="common_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>40</y>
     <width>891</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>common info</string>
   </property>
   <widget class="QWidget" name="layoutWidget_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>179</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_33">
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Base Price:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="base_price_le_3">
       <property name="placeholderText">
        <string>$million$</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_20">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>179</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_34">
     <item>
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>Hole Area:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hole_area_le_3">
       <property name="placeholderText">
        <string>Square meter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Building Mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="building_mode_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>229</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>361</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_35">
     <item>
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address_le_3">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_22">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>95</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QPushButton" name="add_picture_3">
       <property name="text">
        <string>Add Picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Insert_3">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>170</y>
     <width>891</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_41">
    <item>
     <widget class="QGroupBox" name="n_villa_gp_2">
      <property name="title">
       <string>Norther Villa</string>
      </property>
      <widget class="QWidget" name="layoutWidget_23">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>35</y>
         <width>253</width>
         <height>180</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_36">
          <item>
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Built Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="built_area_notrh_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_37">
          <item>
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Number of rooms:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rooms_north_3">
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_38">
          <item>
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>Front yard Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="front_area_north_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_39">
          <item>
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Back yard Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="back_area_north_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="s_villa_gp_2">
      <property name="title">
       <string>Southern Villa</string>
      </property>
      <widget class="QWidget" name="layoutWidget_18">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>36</y>
         <width>253</width>
         <height>180</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_29">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Built Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="built_area_south_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_30">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Number of rooms:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rooms_south_3">
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Yard Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yard_area_south_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_32">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Parking Area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="parking_area_south_3">
            <property name="placeholderText">
             <string>Square meter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="apartment_gp_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Aprtment</string>
      </property>
      <widget class="QPushButton" name="add_house_pb_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Show House</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>261</width>
         <height>60</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_40">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Number of floors:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="floors_ap_3">
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="have_elevator_3">
          <property name="text">
           <string>Have an elevator</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="ap_pictures">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Pctures</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
